content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -310,6 +310,14 @@ Used on: Pods
 
 This annotation allows you to specify the AppArmor security profile for a container within a
 Kubernetes pod. 
+
+Example: `container.apparmor.security.beta.kubernetes.io/defaultProfileName: my-custom-profile`
```

Comment:
This is a duplicate line.

Reply:
What do you mean by duplicate? This is the only occurrence of `container.apparmor.security.beta.kubernetes.io/defaultProfileName` in this file. 

It's below `container.apparmor.security.beta.kubernetes.io/*` right after `container.apparmor.security.beta.kubernetes.io/my-container`. 

Maybe changing the end of annotation key `my-container` to `defaultProfileName` is not enough for readers to notice?

Reply:
Ah, it's not a duplicate, but there's a missing heading.